src/test/java/org/jitsi/rtp/rtp/RtpPacketTest.kt

```diff
@@ -96,15 +102,18 @@ class RtpPacketTest : ShouldSpec() {
                 ext.currExtBuffer.getByteAsInt(ext.currExtOffset + 1) shouldBe 0xFF.toPositiveInt()
 
                 rtpPacket.payloadLength shouldBe dummyRtpPayload.size
+                rtpPacket.getPayload() should haveSameContentAs(UnparsedPacket(dummyRtpPayload))
```

Comment:
this feels a bit weird...we create a packet but just pass in a payload?  that could cause issues if the packet tries to parse part of the header (which doesn't actually exist here)

Reply:
An `UnparsedPacket` shouldn't parse the header, right? I only need a `ByteArrayBuffer` here, but it's abstract

Reply:
no, true, it doesn't, but given we already compared the buffers directly before i guess i didn't see the point.